packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js

```diff
@@ -0,0 +1,361 @@
+import {LightningElement} from 'lwc';
+
+import more from '@salesforce/label/c.quantic_More';
+
+/**
+ *  The `QuanticTabBar` component displays the Quantic Tabs in a responsive manner. When tabs are wider than the available space, the tabs that cannot fit in the space are moved in the "More" drop-down list.
+ * @category Search
+ * @example
+ * <c-quantic-tab-bar>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 1" expression={expressionOne} is-active></c-quantic-tab>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 2" expression={expressionTwo}></c-quantic-tab>
+ *   <c-quantic-tab engine-id={engineId} label="Tab 3" expression={expressionThree}></c-quantic-tab>
+ * </quantic-tab-bar>
+ */
+export default class QuanticTabBar extends LightningElement {
+  labels = {
+    more,
+  };
+
+  /** @type {boolean} */
+  hasRendered = false;
+  /** @type {boolean} */
+  isComboboxOpen = false;
+  /** @type {Array<{value: string, label: string}>} */
+  options = [];
+  /** @type {number} */
+  maxMoreButtonWidth = 0;
+  /** @type {boolean} */
+  expandedMoreButton = true;
+
+  connectedCallback() {
+    window.addEventListener('resize', this.updateTabsDisplay);
+    this.addEventListener('tab_rendered', this.updateTabsDisplay);
+  }
+
+  renderedCallback() {
+    if (!this.hasRendered) {
+      this.hideMoreButton();
+      this.hasRendered = true;
+    }
+  }
+
+  /**
+   * Updates the display of the tabs.
+   * @returns {void}
+   */
+  updateTabsDisplay = () => {
+    if (this.isOverflow) {
+      this.showMoreButton();
+    } else {
+      this.hideMoreButton();
+    }
+    this.updateTabVisibility(this.overflowingTabs, false);
+    this.updateTabVisibility(this.displayedTabs, true);
+    this.updateComboboxOptions();
+    this.updateMoreButtonPosition();
+    this.isComboboxOpen = false;
+  };
+
+  /**
+   * Updates the combobox options.
+   * @returns {void}
+   */
+  updateComboboxOptions() {
+    this.options = this.overflowingTabs.map((el) => ({
+      // @ts-ignore
+      label: el.label,
+      // @ts-ignore
+      value: el.expression,
+    }));
+  }
+
+  /**
+   * Updates the position of the more button element.
+   * @returns {void}
+   */
+  updateMoreButtonPosition() {
+    this.moreButton?.style.setProperty(
```

Comment:
Perhaps it should be explained why you need this in a comment somewhere to explain the logic.

Reply:
We are updating the `more button` position like that because all the tabs are always present inside the tabs container, even the hidden tabs are in there. we have something like this:
__________________________________________________
|________________TABS________________| |___ More ___|
|displayedTab1-displayedTab2-hiddenTab3| |___ More ___|
The hidden tab has a CSS visibility set to `hidden` so the tab will not appear but there is still space allocated for it in the page. 
That's why we update the position of the `more button` relatively to the last displayed tab.

Reply:
Okay interesting, I still think part of the logic should be explained somewhere in a comment :P